### DISCUSSION TITLE
(submod) casual convos (working title)

### DISCUSSION THREAD

**Discussion by Finnthefox24** [Monika-After-Story/MonikaModDev#7821] [general - submod / external content]
[here's the link to the submod](https://github.com/Finnthefox24/submod-test)

Short conversations, songs, fun facts, and moods.

**Comment by Finnthefox24** [comment]
I will now disappear from the surface of earth for 2-4 business days.

**Comment by xXMidnight** [comment]
yEs thAnk yOU yOU hAvE mAdE mY dAY

uh... where do I put it
